lib/initializer.rb

```diff
@@ -0,0 +1,56 @@
+require_relative './game.rb'
+require_relative './board/board.rb'
+require_relative './board/board_checker.rb'
+require_relative './players/ai.rb'
+require_relative './players/human.rb'
+require_relative './players/player.rb'
+require_relative './ui/user_interface.rb'
+
+class Initializer
+  attr_accessor :logic1
+
+  def initialize(size, empty_char)
+    @size = size
+    @empty_char = empty_char
+    @player1
+    @player2
+  end
+
+  def start
+    board = Board.new(@size, @empty_char)
+    checker = BoardChecker.new(@size, @empty_char)
+    user_interface = UserInterface.new(board, 'X', 'O')
+    ai_logic = AI.new(checker, 'O', 'X')
+    human_logic = Human.new(board, user_interface, 'X')
+
+    welcome(user_interface)
+    selection = who_goes_first?(user_interface)
+    setup_players(selection, ai_logic, human_logic, user_interface)
+
+    game = Game.new(board, checker, @player1, @player2)
+    play(game)
```

Comment:
Ok, I tried to change the initializer to a `main` kind of method.  Let me know if it is better or worse.

[Initializer method](https://github.com/saralein/tic-tac-toe_ruby/pull/7/files#diff-6bfad8daa181b5e0a43f92245aa59e75R10)